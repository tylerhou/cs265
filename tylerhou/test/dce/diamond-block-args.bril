@main {
  cond: bool = const true;
  br cond .left .right;
.left:
  a: int = const 1;
  b: int = const 1;
  jmp .end(a);
.right:
  a: int = const 2;
  b: int = const 2;
  jmp .end(b);
.end(c: int):
  print c;
}
