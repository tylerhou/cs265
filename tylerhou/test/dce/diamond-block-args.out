@main {
.__b1:
  cond: bool = const true;
  br cond .left .right;
.left:
  a: int = const 1;
  jmp .end(a);
.right:
  b: int = const 2;
  jmp .end(b);
.end(c: int):
  print c;
  ret;
}
